spark/v3.2/build.gradle

```diff
@@ -31,10 +30,7 @@ configure(sparkProjects) {
   configurations {
     all {
       resolutionStrategy {
-        force "com.fasterxml.jackson.module:jackson-module-scala_${scalaVersion}:2.12.3"
-        force 'com.fasterxml.jackson.module:jackson-module-paranamer:2.12.3'
-        force 'com.fasterxml.jackson.core:jackson-databind:2.12.3'
-        force 'com.fasterxml.jackson.core:jackson-core:2.12.3'
+        force "com.fasterxml.jackson.module:jackson-module-scala_${scalaVersion}:${libs.versions.jackson212.get()}"
```

Comment:
same question as for Spark 3.1

Reply:
PR was opened for so long that I forgot the reason why I did that in the first place, I reverted it back